components/ActionSheet/tests/snapshot/MDCActionSheetControllerSnapshotTests.m

```diff
@@ -871,4 +871,55 @@ - (void)testThreeActionsSufficientSizeShortTextLTRWithDefaultPresentationStyleOn
   [self snapshotVerifyViewForIOS13:window];
 }
 
+- (void)testActionSheetWithHeaderShown {
+  // Given
+  MDCActionSheetAction *action1 =
+      [MDCActionSheetAction actionWithTitle:kShortTitle1Latin
+                                      image:[UIImage mdc_testImageOfSize:CGSizeMake(24, 24)]
+                                    handler:nil];
+  MDCActionSheetAction *action2 =
+      [MDCActionSheetAction actionWithTitle:kShortTitle2Latin
+                                      image:[UIImage mdc_testImageOfSize:CGSizeMake(24, 24)]
+                                    handler:nil];
+  self.actionSheetController = [MDCActionSheetController actionSheetControllerWithTitle:@"Foo"];
+  [self.actionSheetController addAction:action1];
+  [self.actionSheetController addAction:action2];
+  self.actionSheetController.view.bounds = CGRectMake(0, 0, 320, 200);
+
+  // When
+  self.actionSheetController.showsHeaderDivider = YES;
+  self.actionSheetController.headerDividerColor = UIColor.blackColor;
+  [self.actionSheetController.view setNeedsLayout];
+  [self.actionSheetController.view layoutIfNeeded];
+
+  // Then
+  [self generateSnapshotAndVerifyForView:self.actionSheetController.view];
+}
+
+- (void)testActionSheetWithHeaderShownButNoTitleOrMessage {
```

Comment:
Looks like the golden image for this test needs updating.

Reply:
done